Architecture isn't supported: armv8l

LXD 5.0.0 has been uploaded to the Debian archive for the first time, and is hitting the various buildds. Builds for both the `armel` and `armhf` architectures compile fine, but several of the tests fail with the error "Architecture isn't supported: armv8l". For example,

```
=== RUN   TestIntegration_UnixSocket
    testing.go:37: 
        	Error Trace:	/<<PKGBUILDDIR>>/_build/src/github.com/lxc/lxd/lxd/testing.go:37
        	            				/<<PKGBUILDDIR>>/_build/src/github.com/lxc/lxd/lxd/daemon_integration_test.go:36
        	            				/<<PKGBUILDDIR>>/_build/src/github.com/lxc/lxd/lxd/daemon_integration_test.go:19
        	Error:      	Received unexpected error:
        	            	Architecture isn't supported: armv8l
        	Test:       	TestIntegration_UnixSocket
--- FAIL: TestIntegration_UnixSocket (0.00s)
```

It looks like the logic for testing supported platforms is in `shared/osarch/architectures.go`. I'm not super familiar with all the different ways arm architectures are identified, but given that LXD compiled and lots of other tests pass during each build, I hope it won't be too difficult extend the supported architecture matrix so the tests can be run on these two additional architectures.

I am working on uploading LXD 5.0.1, but I don't think that will have any affect on this issue.

Buildd logs:
https://buildd.debian.org/status/fetch.php?pkg=lxd&arch=armel&ver=5.0.0-1&stamp=1662903087&raw=0
https://buildd.debian.org/status/fetch.php?pkg=lxd&arch=armhf&ver=5.0.0-1&stamp=1662902755&raw=0

--- a/shared/osarch/architectures.go
+++ b/shared/osarch/architectures.go
@@ -18,12 +18,16 @@ const (
 	ARCH_64BIT_MIPS                  = 10
 	ARCH_32BIT_RISCV_LITTLE_ENDIAN   = 11
 	ARCH_64BIT_RISCV_LITTLE_ENDIAN   = 12
+	ARCH_32BIT_ARMV6_LITTLE_ENDIAN   = 13
+	ARCH_32BIT_ARMV8_LITTLE_ENDIAN   = 14
 )
 
 var architectureNames = map[int]string{
 	ARCH_32BIT_INTEL_X86:             "i686",
 	ARCH_64BIT_INTEL_X86:             "x86_64",
+	ARCH_32BIT_ARMV6_LITTLE_ENDIAN:   "armv6l",
 	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   "armv7l",
+	ARCH_32BIT_ARMV8_LITTLE_ENDIAN:   "armv8l",
 	ARCH_64BIT_ARMV8_LITTLE_ENDIAN:   "aarch64",
 	ARCH_32BIT_POWERPC_BIG_ENDIAN:    "ppc",
 	ARCH_64BIT_POWERPC_BIG_ENDIAN:    "ppc64",
@@ -38,7 +42,9 @@ var architectureNames = map[int]string{
 var architectureAliases = map[int][]string{
 	ARCH_32BIT_INTEL_X86:             {"i386", "i586", "386", "x86", "generic_32"},
 	ARCH_64BIT_INTEL_X86:             {"amd64", "generic_64"},
-	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   {"armel", "armhf", "arm", "armhfp", "armv7a_hardfp", "armv7", "armv7a_vfpv3_hardfp"},
+	ARCH_32BIT_ARMV6_LITTLE_ENDIAN:   {"armel", "arm"},
+	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   {"armhf", "armhfp", "armv7a_hardfp", "armv7", "armv7a_vfpv3_hardfp"},
+	ARCH_32BIT_ARMV8_LITTLE_ENDIAN:   {},
 	ARCH_64BIT_ARMV8_LITTLE_ENDIAN:   {"arm64", "arm64_generic"},
 	ARCH_32BIT_POWERPC_BIG_ENDIAN:    {"powerpc"},
 	ARCH_64BIT_POWERPC_BIG_ENDIAN:    {"powerpc64", "ppc64"},
@@ -52,7 +58,9 @@ var architectureAliases = map[int][]string{
 var architecturePersonalities = map[int]string{
 	ARCH_32BIT_INTEL_X86:             "linux32",
 	ARCH_64BIT_INTEL_X86:             "linux64",
+	ARCH_32BIT_ARMV6_LITTLE_ENDIAN:   "linux32",
 	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   "linux32",
+	ARCH_32BIT_ARMV8_LITTLE_ENDIAN:   "linux32",
 	ARCH_64BIT_ARMV8_LITTLE_ENDIAN:   "linux64",
 	ARCH_32BIT_POWERPC_BIG_ENDIAN:    "linux32",
 	ARCH_64BIT_POWERPC_BIG_ENDIAN:    "linux64",
@@ -67,8 +75,10 @@ var architecturePersonalities = map[int]string{
 var architectureSupportedPersonalities = map[int][]int{
 	ARCH_32BIT_INTEL_X86:             {},
 	ARCH_64BIT_INTEL_X86:             {ARCH_32BIT_INTEL_X86},
-	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   {},
-	ARCH_64BIT_ARMV8_LITTLE_ENDIAN:   {ARCH_32BIT_ARMV7_LITTLE_ENDIAN},
+	ARCH_32BIT_ARMV6_LITTLE_ENDIAN:   {},
+	ARCH_32BIT_ARMV7_LITTLE_ENDIAN:   {ARCH_32BIT_ARMV6_LITTLE_ENDIAN},
+	ARCH_32BIT_ARMV8_LITTLE_ENDIAN:   {ARCH_32BIT_ARMV6_LITTLE_ENDIAN, ARCH_32BIT_ARMV7_LITTLE_ENDIAN},
+	ARCH_64BIT_ARMV8_LITTLE_ENDIAN:   {ARCH_32BIT_ARMV6_LITTLE_ENDIAN, ARCH_32BIT_ARMV7_LITTLE_ENDIAN, ARCH_32BIT_ARMV8_LITTLE_ENDIAN},
 	ARCH_32BIT_POWERPC_BIG_ENDIAN:    {},
 	ARCH_64BIT_POWERPC_BIG_ENDIAN:    {ARCH_32BIT_POWERPC_BIG_ENDIAN},
 	ARCH_64BIT_POWERPC_LITTLE_ENDIAN: {},

